### PR TITLE
Loading private profiles properly

### PR DESCRIPTION
**Pull request by 0xWerz**
#133 
## Preview:

image

## Inspired from the 2015 UI:
image

### CODE REVIEW

File: layouts/profile/script.js [hunk 1]
```diff
@@ -593,7 +608,7 @@ async function renderProfile() {
     document.getElementById('profile-stat-followers-value').innerText = Number(pageUser.followers_count).toLocaleString().replace(/\s/g, ',');
     document.getElementById('profile-stat-favorites-value').innerText = Number(pageUser.favourites_count).toLocaleString().replace(/\s/g, ',');
 
-    document.getElementById('tweet-nav').hidden = pageUser.statuses_count === 0 || !(subpage === 'profile' || subpage === 'replies' || subpage === 'media');
+    document.getElementById('tweet-nav').hidden = pageUser.statuskes_count === 0 || user_protected || !(subpage === 'profile' || subpage === 'replies' || subpage === 'media');
```
Comment (dimdenGD) [inline layouts/profile/script.js:611]: Typo

File: layouts/profile/script.js [hunk 2]
```diff
@@ -294,19 +300,28 @@ async function updateTimeline() {
...
+                    tweetsCursor = tl.cursor;
+                    tl = tl.tweets;
+                }
             } else {
-                tl = await API.getUserTweetsV2(pageUser.id_str, undefined, subpage !== 'profile');
-                tweetsCursor = tl.cursor;
-                tl = tl.tweets;
+                document.getElementById(
+                    "timeline"
+                ).innerHTML = `<div style="padding: 50px;color: var(--darker-gray);"><h1> Tweets are protected </h1> <p> This account's Tweets are protected. Only confirmed followers have access to @${pageUser.screen_name}'s Tweets and complete profile. <br/> Click the "Follow" button to send a follow request.</p></div>`;
```
Comment (dimdenGD) [inline layouts/profile/script.js]: Needs to use locale files instead of text in code

**Review by dimdenGD — CHANGES_REQUESTED** [review]
Does this support being followed to protected user? You should be allowed to see tweets if you are

File: layouts/profile/script.js (hunk 2 above)
Comment (0xWerz) [reply layouts/profile/script.js]: Should be good now.

**Comment by dimdenGD** [comment]
This doesn't follow proper format of locale files. Every line can only have `message`, `description` and `example` fields (in this case it's just `message` with multiple different lines.  Also you shouldn't use Google Translate or whatever tool you used, only add English and if you know any other language you can translate it yourself.

**Comment by 0xWerz** [comment]
I think the description field is necessary; there are two blocks of text, the title and the bottom description, which are different in font and size. What do you think?

**Comment by dimdenGD** [comment]
"Description" field is used to help translators give context, not to be used in code itself. Just make another line for text you have inside "description" and put it inside "message". Also please move new lines to the bottom of locale file so it can be easily seen as new thing to translate.

**Comment by 0xWerz** [comment]
I have simply replicated what you've done with the `hasnt_tweeted` field.

```js
document.getElementById('no-tweets').innerHTML = `
          <h3>${LOC.hasnt_tweeted.message.replace('$SCREEN_NAME$', `<span>${pageUser.screen_name}</span>`)}</h3>
          <p>${LOC.when_theyll_tweet.message}</p>
        `;
```

```json

"hasnt_tweeted": { "message": "@$SCREEN_NAME$ hasn't tweeted", "placeholders": { "screen_name": { "content": "loading" } } },
"when_theyll_tweet": { "message": "When they do, their Tweets will show up here." },
```

**Comment by dimdenGD** [comment]
Thank you!

**Pull request merged**